src/Handle.jsx

```diff
@@ -1,22 +1,20 @@
 import React, { PropTypes } from 'react';
+import assign from 'object-assign';
 
 export default class Handle extends React.Component {
   render() {
     const {
-      className, vertical, offset, minimumTrackTintColor, disabled, ...restProps,
+      className, vertical, offset, handleStyle, disabled, ...restProps,
     } = this.props;
     const style = vertical ? { bottom: `${offset}%` } : { left: `${offset}%` };
-    if (minimumTrackTintColor && !disabled) {
-      style.borderColor = minimumTrackTintColor;
-    }
-    return <div {...restProps} className={className} style={style} />;
+    return <div {...restProps} className={className} style={assign({}, style, handleStyle)} />;
```

Comment:
@STRML fixed in https://github.com/react-component/slider/commit/441779cdcb1022685ac13a212b11d589176513e8